src/behaviors/Behavior.js

```diff
@@ -721,13 +717,13 @@ var Behavior = Base.extend('Behavior', {
      * @memberOf Behavior.prototype
      * @desc update the data at point x, y with value
      * @return The data.
-     * @param {number} x - x coordinate
-     * @param {number} y - y coordinate
+     * @param {number} c - Grid column coordinate.
+     * @param {number} r - Grid row coordinate.
      * @param {Object} value - the value to use
      */
-    setValue: function(x, y, value) {
-        var column = this.getActiveColumn(x);
-        return column && column.setValue(y, value);
+    setValue: function(c, r, value) {
```

Comment:
I don't like it either but it's so fundamental I think that at this point it really would screw people up to change it. `r` is appropriate (we do need cell renderers for header row cells) but not sure why it is `x` and not `c`.